sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -217,8 +217,11 @@ object FileFormatWriter extends Logging {
 
       val commitMsgs = ret.map(_.commitMsg)
 
+      val commitJobStartTime = System.nanoTime()
+      logInfo(s"Start to commit write Job ${description.uuid}.")
       committer.commitJob(job, commitMsgs)
-      logInfo(s"Write Job ${description.uuid} committed.")
+      logInfo(s"Write Job ${description.uuid} committed. " +
+        s"Cost time: ${(System.nanoTime() - commitJobStartTime) / 1000 / 1000} ms")
```

Comment:
Maybe should assign `System.nanoTime()` to a `val commitJobEndTime` first, otherwise the `Cost time` may not be accurate.

Reply:
yea, that's more accurate.